packages/react-ui/components/Button/Button.tsx

```diff
@@ -13,7 +13,7 @@ import { Corners } from './Corners';
 
 export type ButtonSize = 'small' | 'medium' | 'large';
 export type ButtonType = 'button' | 'submit' | 'reset';
-export type ButtonUse = 'default' | 'primary' | 'success' | 'danger' | 'pay' | 'link';
+export type ButtonUse = 'default' | 'primary' | 'success' | 'danger' | 'pay' |'success'| 'link';
```

Comment:
```suggestion
export type ButtonUse = 'default' | 'primary' | 'success' | 'danger' | 'pay' | 'link';
```

Тут `success` не нужно добавлять, он уже есть.